reconcile/service_dependencies.py

```diff
@@ -1,54 +1,20 @@
 import sys
 import logging
+from typing import Any, Mapping
+from reconcile.gql_queries.service_dependencies import service_dependencies
+from reconcile.gql_queries.service_dependencies.service_dependencies import (
+    AppV1,
+    ServiceDependenciesQuery,
+)
 
 from reconcile.utils import gql
 from reconcile import queries
-from reconcile.utils.external_resources import (
-    get_provision_providers,
```

Comment:
`get_provision_providers` seems to not be used anywhere anymore (except in a test file), can we remove it and clean the code further ?